hack/generator/pkg/astmodel/armconversion/convert_from_arm_function_builder.go

```diff
@@ -218,6 +219,23 @@ func (builder *convertFromARMBuilder) ownerPropertyHandler(
 	return []dst.Stmt{result}
 }
 
+// conditionsPropertyHandler generates conversions the "Conditions" status property. This property is set by the controller
+// after each reconcile and so currently is not preserved.
+func (builder *convertFromARMBuilder) conditionsPropertyHandler(
+	toProp *astmodel.PropertyDefinition,
+	_ *astmodel.ObjectType) []dst.Stmt {
+
+	if toProp.PropertyName() != builder.idFactory.CreatePropertyName(astmodel.ConditionsProperty, astmodel.Exported) || builder.typeKind != TypeKindStatus {
+		return nil
+	}
+
+	// Returning an empty statement allows us to "consume" this match and not proceed to the next handler.
```

Comment:
Returning an empty statement feels like a bit of a hack, esp given it's not idiomatic Go to return sentinel values. Consider a later refactoring where each handler returns `([]dst.Stmt, bool)`?

Reply:
I've made a note to do this but will do it in a later PR. Given that the infrastructure that uses this function signature is shared across to/from ARM (and there are a number of methods in each) it'll be a pretty big refactor.

Agree it'd make it a bit clearer though.